core/src/test/java/tachyon/client/FileOutStreamTest.java

```diff
@@ -48,7 +48,7 @@ public static final void afterClass() throws Exception {
 
   @BeforeClass
   public static final void beforeClass() throws IOException {
-    System.setProperty("tachyon.user.quota.unit.bytes", "1000");
```

Comment:
In this PR, Client side doesn't maintain space anymore, when writing a block file, client should request space for the block file at least one time. and the space requested can be only used for current block file, the request but not used space will be reclaimed when caching block.

when writing a file, all blocks of the file will be cached when the file is closed, which means that the request but not used space can only be reclaimed when the closing the file.

In this test case, if setting tachyon.user.quota.unit.bytes to 1000, which means that client will request at least 1000 bytes for each block file. and it will run out of space before the file is finished.